src/azure-cli/azure/cli/command_modules/appconfig/custom.py

```diff
@@ -243,6 +244,44 @@ def regenerate_credential(cmd, client, name, id_, resource_group_name=None):
     return client.regenerate_key(resource_group_name, name, RegenerateKeyParameters(id=id_))
 
 
+def list_replica(cmd, client, store_name, resource_group_name=None):
+    if resource_group_name is None:
+        resource_group_name, _ = resolve_store_metadata(cmd, store_name)
+
+    return client.list_by_configuration_store(resource_group_name=resource_group_name, config_store_name=store_name)
+
+
+def show_replica(cmd, client, store_name, name, resource_group_name=None):
+    if resource_group_name is None:
+        resource_group_name, _ = resolve_store_metadata(cmd, store_name)
+    try:
+        return client.get(resource_group_name=resource_group_name, config_store_name=store_name, replica_name=name)
+    except ResourceNotFoundError:
+        # Show a meaningful error message than the one coming from the server.
+        raise ResourceNotFoundError("The replica '{}' for App Configuration '{}' not found.".format(name, store_name))
+
+
+def create_replica(cmd, client, store_name, name, location, resource_group_name=None):
+    if resource_group_name is None:
+        resource_group_name, _ = resolve_store_metadata(cmd, store_name)
+
+    replica_creation_params = Replica(location=location)
+    return client.begin_create(resource_group_name=resource_group_name,
+                               config_store_name=store_name,
```

Comment:
Any known errors create method may throw? Like in the ```show_replica``` method, do you also want to customize some error message? Usually we will use whatever the server returns.

Reply:
The `ResourceNotFoundError` in `show_replica` is a special case where the error message returned by the server isn't clear, that's the reason I made an exception to override the server's error message with a meaningful one.
Other errors at other places return clear error messages so we don't need to override those error messages.

Reply:
Okay, that makes sense.